modules/organism.js

```diff
@@ -10,6 +10,7 @@ function Organism(generation, mass, x, y, visionRange, visionAngle, minSpeed, ma
   this.energy = 0.5 * this.mass * this.maxSpeed * this.maxSpeed;
   this.age = 1;
   this.mature = false;
+  this.blind = false;
```

Comment:
I dont think this should be part of an organism attribute. This should be a global variable.